nautobot/dcim/forms.py

```diff
@@ -4224,11 +4224,14 @@ class Meta:
 class ControllerDeviceGroupForm(NautobotModelForm):
     """ControllerDeviceGroup create/edit form."""
 
+    devices = DynamicModelMultipleChoiceField(queryset=Device.objects.all(), required=False)
```

Comment:
Doesn't update devices.

Any clue how to fix this or link to the same pattern pls.? @whitej6 @glennmatthews

Reply:
Works nice, TY!